src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -6302,6 +6304,10 @@ private static void processNode(final TagNode node, final TagStatus status) {
         FightRequest.handleTrainbotLuggageHook(str, status);
       }
 
+      if (won && status.armtowel) {
```

Comment:
I think there's now enough of these `won` blocks that I'd prefer to see:
```
if (won) {
  if (status.X) {
  [...]
}
```

Reply:
I had thought about that and I agree.
I could also put the status.xxx checks into the individual methods, but I am intentionally not doong method calls unless necessary.
Similarly, I could even put all three into a single method, but I prefer having each method be specific to one thing.